nemo/collections/llm/modelopt/quantization/utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from pathlib import Path
+from typing import Any, Dict, Union
+
+from nemo.utils.cast_utils import maybe_cast_to_type
+
+
+def standardize_json_config(quant_cfg: Dict[str, Any]):
+    """Standardize the quantization configuration loaded from a JSON file to
+    ensure compatibility with modelopt. Modifiy the input dictionary in place.
+
+    Args:
+        quant_cfg (Dict[str, Any]): The quantization config dictionary to standardize.
+    """
+    for key, value in quant_cfg.items():
+        if key == "block_sizes":
+            value = {maybe_cast_to_type(k, int): v for k, v in value.items()}
```

Comment:
should `v` also be cast to int or only `k`?

Reply:
When a config is loaded from a JSON file, all keys are loaded as strings, but values load correctly as ints.